include/sdf/sdf.hh

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef SDF_SDF_HH_
+#define SDF_SDF_HH_
+
+// TODO(anyone): deprecate this header?
```

Comment:
@chapulina mentioned the other day/week that according to the google c++ style guide, the name in a `TODO` should actually be the name of the person who created it, since this person knows the most about the `TODO`: https://google.github.io/styleguide/cppguide.html#TODO_Comments

I know that in the past, we've been making the name in a `TODO` the name of someone who should fix it, but we should change this to adhere more closely to the style guide - @scpeters, would you mind putting your name in the `TODO` since you're the one who created this `TODO` and knows the most about it?

Reply:
I copied it from https://github.com/ignitionrobotics/sdformat/pull/747, but I'll just remove the comment since it's not strictly necessary

Reply:
removed the comment in https://github.com/ignitionrobotics/sdformat/pull/780/commits/a13f9e5e65f8f941619eb9dd41324bad89981875